jobs/build_ova/build_ova.groovy

```diff
@@ -6,53 +6,55 @@ node{
     shareMethod = load("jobs/ShareMethod.groovy")
 }
 
-String label_name = "packer_ova"
-lock(label:label_name,quantity:1){
-    resources_name = shareMethod.getLockedResourceName(label_name)
-    if(resources_name.size>0){
-        node_name = resources_name[0]
-    }
-    else{
-        error("Failed to find resource with label " + label_name)
-    }
-    node(node_name){ws{
-        timestamps{
-            withEnv([
-                "RACKHD_COMMIT=${env.RACKHD_COMMIT}",
-                "RACKHD_VERSION=${env.RACKHD_VERSION}",
-                "IS_OFFICIAL_RELEASE=${env.IS_OFFICIAL_RELEASE}",
-                "OS_VER=${env.OS_VER}",
-                "BUILD_TYPE=vmware", 
-                "BINTRAY_SUBJECT=${env.BINTRAY_SUBJECT}",
-                "BINTRAY_REPO=debian",
-                "CI_BINTRAY_SUBJECT=${env.CI_BINTRAY_SUBJECT}",
-                "CI_BINTRAY_REPO=debian", 
-                "BINTRAY_COMPONENT=main", 
-                "BINTRAY_DISTRIBUTION=trusty", 
-                "BINTRAY_ARCHITECTURE=amd64"]){
-                def current_workspace = pwd()
-                deleteDir()
-                dir("on-build-config"){
-                    checkout scm
-                }
-                def url = "https://github.com/RackHD/RackHD.git"
-                def branch = "${env.RACKHD_COMMIT}"
-                def targetDir = "build"
-                shareMethod.checkout(url, branch, targetDir)
-                step ([$class: 'CopyArtifact',
-                projectName: 'PACKER_CACHE_BUILD',
-                target: 'cache_image']);
+lock("ova_build"){
```

Comment:
talk with @PengTian0 
there's re 2 options of the locking . and option 1 was what I intended. but she chose option 2

per @PengTian0 , 
option 1:
Pros : smaller lock , only lock when necessary. reasonable code logic.
Cons:  when a ova-build job has locked slave, but waiting resource lock(cache build ongoing). But the slave will be idle. and there's 'ova-post-test' requires same slaves will wait. 
option 2:
Pros : only lock slave if cache is ready( resource lock acquired)
Cons : seems bigger lock scope 


so Tian think make slave used more effective is of more priority



option 1:
```
lock_slave()
lock_resource()  // the lock between OVA_CACHE_BUILD & OVA_BUILD
copy_from_cache_archiving()
unlock_resource()
do_ova_build()
unlock_slave()
```

option 2:
```
lock_resource()  // the lock between OVA_CACHE_BUILD & OVA_BUILD
lock_slave()
copy_from_cache_archiving()
do_ova_build()
unlock_slave()
unlock_resource()
```